packages/compiler/src/server/tspconfig/completion.ts

```diff
@@ -80,6 +108,74 @@ export async function provideTspconfigCompletionItems(
         itemsFromEmitter.push(...more);
       }
       return [...itemsFromBuiltIn, ...itemsFromEmitter];
+    } else if (nodePath.length > CONFIG_PATH_LENGTH_FOR_LINTER_LIST && nodePath[0] === "linter") {
+      const linterName = nodePath[CONFIG_PATH_LENGTH_FOR_LINTER_LIST - 1];
+      const items: CompletionItem[] = [];
+      const linters = await linterProvider.listLibraries(tspConfigFile);
+
+      for (const [name, pkg] of Object.entries(linters)) {
+        if (linterName === "extends") {
```

Comment:
put the if outside of for, you dont need to check it everytime, and have branches first will also make the logic more independent and clean